<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-KQUFIDC" name="com.qa.hubspot.tests.MeetingsChatPageTest" tests="3" failures="0" timestamp="2020-07-15T11:32:20 EDT" time="0.000" errors="1">
  <testcase name="verifyGetTitleOFChatPageHeaderTest" time="0.000" classname="com.qa.hubspot.tests.MeetingsChatPageTest">
    <skipped/>
  </testcase> <!-- verifyGetTitleOFChatPageHeaderTest -->
  <system-out/>
  <testcase name="meetingChatPageSetUp" time="0.000" classname="com.qa.hubspot.tests.MeetingsChatPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.tests.MeetingsChatPageTest.meetingChatPageSetUp(MeetingsChatPageTest.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- meetingChatPageSetUp -->
  <system-out/>
  <testcase name="verifyGetTitleOFChatPageTest" classname="com.qa.hubspot.tests.MeetingsChatPageTest">
    <skipped/>
  </testcase> <!-- verifyGetTitleOFChatPageTest -->
  <system-out/>
</testsuite> <!-- com.qa.hubspot.tests.MeetingsChatPageTest -->
